Bio/PDB/Entity.py

```diff
@@ -435,6 +462,37 @@ def get_id(self):
         """Return the id."""
         return self.id
 
+    def strictly_equals(self, other: "DisorderedEntityWrapper") -> bool:
+        """Compare this entity to the other entity using a strict definition of equality.
+
+        Recursively compare the children of this entity to the other entity's children.
+        Compare most properties including names and atomic coordinates.
+
+        :param other: The entity to compare this entity with
+        :type other: DisorderedEntityWrapper
+        :return: Whether the two entities are strictly equal
+        :rtype: bool
+        """
+        if not isinstance(other, type(self)):
+            return False
+
+        if self.id != other.id:
+            return False
+
+        if not self.selected_child.get_id() == other.selected_child.get_id():
+            return False
```

Comment:
I added this in my latest push. Disordered entities have to have the same selected child to be considered strictly equal.